.gitignore

```diff
@@ -10,3 +10,4 @@ remote_debugger_settings.py
 
 # Test data
 test/data
+remote_debugger_settings.py
```

Comment:
Is denk ik niet nodig. Dit bestand is toch al gecommit, en het misschien wel handig voor andere PyCharm gebruikers later.